### ISSUE TITLE
metadata.rb for 8.0.0 references a gem that cannot be found

### ISSUE THREAD

**Issue by fletchowns** [chef-cookbooks/aws#368]
My converge is failing after upgrading to version 8.0.0 of this cookbook with:
```
Could not find gem 'aws-sdk-kms-core (~> 1.0.0)' in any of the gem sources
```

There's a reference to this gem in the metadata.rb: https://github.com/chef-cookbooks/aws/blob/48263b126d3ba3e4080f42c3caf16cbd3754800e/metadata.rb#L31 

Searching for it on rubygems.org I can't find it: https://rubygems.org/search?utf8=%E2%9C%93&query=aws-sdk-kms-core

**Comment by majormoses** [comment]
That's odd I had been running this version in my environments for several days without issue. I will see about getting a PR for you to test.

**Comment by majormoses** [comment]
@fletchowns apologies I think it should be fixed with #369 not sure how it got past travis and worked in my envs but I confirmed that the gem was copy/pasted wrongly.

**Comment by fletchowns** [comment]
No worries @majormoses ! Thanks for the lightning fast response and fix!
